### PR TITLE
Write up on Ember CLI improvement tips

### PR DESCRIPTION
**Pull request by abhilashlr**

### CODE REVIEW

File: source/2020-04-03-the-ember-times-issue-142.md [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-## [Section title in sentence case 🐹](#section-url)
+## [Ember CLI improvement tips 🚀](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
-
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[Vasanth](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a Github gist. He talks about a bunch of improvements that include speeding up build times for development and test environments while retaining the right configurations for production. He also added tips on optimising applications using [Mirage](https://www.ember-cli-mirage.com/), but no **spoilers** here, [head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!
```
Comment (amyrlam) [inline source/2020-04-03-the-ember-times-issue-142.md]:
```suggestion
[Vasanth](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a Github gist. He showcases improvements to the `ember-cli-build.js` that speed up build times for development and test environments, while retaining the right configurations for production. He also added tips on optimising applications using [Mirage](https://www.ember-cli-mirage.com/). [Head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!```

**Review by amyrlam — APPROVED** [review]
👍 looks good, thanks for your help this week!

made some suggestions, my reasoning was that it's a gist and not an article per se

**Comment by abhilashlr** [comment]
> 👍 looks good, thanks for your help this week!
> 
> made some suggestions, my reasoning was that it's a gist and not an article per se

Done @amyrlam

File: source/2020-04-03-the-ember-times-issue-142.md [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
-## [Section title in sentence case 🐹](#section-url)
+## [Ember CLI improvement tips 🚀](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
-
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[Vasanth](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a Github gist. He showcases improvements to the `ember-cli-build.js` that speed up build times for development and test environments, while retaining the right configurations for production. He also added tips on optimising applications using [Mirage](https://www.ember-cli-mirage.com/). [Head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!```
```
Comment (chrisrng) [inline source/2020-04-03-the-ember-times-issue-142.md]:
```suggestion
[Vasanth (@vasind)](https://github.com/vasind) collated a list of all [**Ember CLI**](https://ember-cli.com/) improvements for Ember applications as a GitHub gist. He showcased improvements to the `ember-cli-build.js` that speed up build times for development and test environments, while retaining the right configurations in production. He also added tips on optimizing applications using [Mirage](https://www.ember-cli-mirage.com/). [Head out to his gist](https://gist.github.com/vasind/22122632be15f3bc98a092ab012524e2) to check them out!
```

**Review by chrisrng — COMMENTED** [review]
Just minor changes in wording but lgtm thank you!

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for another :sparkles: writeup for the Ember Times!

**Comment by abhilashlr** [comment]
> This reads great! Thank you for another ✨ writeup for the Ember Times!

Thanks to you, @amyrlam and @chrisrng 👍

**Pull request merged**